### PR TITLE
[DE-1047]: updating pandas version due conflict with ds-tools

### PR DESCRIPTION
**Pull request by ramonmarco**
### Where

https://coverwallet.atlassian.net/browse/DE-1047

### CODE REVIEW

**Review by samgans — APPROVED** [review]
LGTM.

File: tests/core/test_dataframe_to_postgre.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 load_dotenv(find_dotenv())
 
 df = DataFrame(columns=["id", "name", "quantity"], 
-    data=[[245, 'Peter', 31.2], [541, 'Lucas', np.nan]])
```
Comment (carlosmagan) [inline tests/core/test_dataframe_to_postgre.py:13]: This is a test case on how nulls are handled, this can't be removed

Comment (ramonmarco) [reply tests/core/test_dataframe_to_postgre.py:13]: added 2 more tests

Comment (ramonmarco) [reply tests/core/test_dataframe_to_postgre.py:13]: Added 2 more tests

**Review by carlosmagan — APPROVED** [review]

**Pull request merged**